includes/HorizontalMirrorAnimation.hpp

```diff
@@ -147,6 +147,8 @@ class HorizontalMirrorAnimation : public LevelAnimation
        value is 0 */
     unsigned short destinationCell {0};
 
+    /* used during the animation; represents the alpha value for the cells
+       of one side of the floor */
     float gradient {255.f};
```

Comment:
a lot of attributes are in both the horizontal and vertical animation, should be refactored